prog/pwm/fancontrol

```diff
@@ -390,6 +393,39 @@ function pwmdisable()
 		return 0
 	fi
 
+	# Try to restore pwmN and pwmN_enable value to the same state as before fancontrol start
+	# restoring the pwmN value is tried first, before the pwmN_enable mode switch. Some chips seem to need this to properly restore fan operation, when activating automatic (2) mode.
+	if [ -v PWMORIGVALUES[${1}] ]
```

Comment:
-v is a bash extension, yet you don't use [[ ]]. It might be better to use something like '[ -n "PWMORIGVALUES[${1}" ]'.